go.mod

```diff
@@ -162,6 +161,4 @@ require (
 	github.com/google/gnostic v0.5.7-v3refs // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/pelletier/go-toml/v2 v2.0.1 // indirect
-	go4.org/intern v0.0.0-20211027215823-ae77deb06f29 // indirect
-	go4.org/unsafe/assume-no-moving-gc v0.0.0-20220617031537-928513b29760 // indirect
```

Comment:
That one sounds like fun :)

Reply:
lol

>If your Go package wants to declare that it plays unsafe games that only work if the Go runtime's garbage collector is not a moving collector, then add:
>
> `import _ "go4.org/unsafe/assume-no-moving-gc"`
>
> Then your program will explode if that's no longer the case. (Users can override the explosion with a scary sounding environment variable.)

Reply:
`
There is no API.
`
From the [godoc](https://pkg.go.dev/go4.org/unsafe/assume-no-moving-gc#section-readme)

Reply:
Probably good that we're not using it anymore